NimbleExtension/Sources/Extensions/Foundation/Collection/Array/Array+OperationOverloading.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Array+OperationOverloading.swift
+//  NimbleExtension
+//
+//  Created by Issarapong Poesua on 8/27/19.
+//  Copyright © 2019 Nimble. All rights reserved.
+//
+
+import Foundation
+
+extension Array {
+    
+    static func += (elements: inout [Element], element: Element) {
```

Comment:
This may be confused.
I just think that will plus two array. Like this ex:
```swift
let strings: [String] = []
strings += ["hello", "world"]
```

I prefer `append`

Reply:
how about we support both of them ? 
```swift

static func += (elements: inout [Element], element: Element)
```

Reply:
:+1: Better, then don't use `append` 🕺